### PR TITLE
[Feature] Support return bitmp/hll data in select statement

### PR DESCRIPTION
**Pull request by yangzhg**
## Proposed changes

Support return bitmp/hll data in select statement, this can be used when set show_object_data=true;

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
- [ ] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [ ] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [x] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/runtime/mysql_result_writer.cpp [hunk 1]
```diff
@@ -115,7 +115,18 @@ int MysqlResultWriter::_add_row_value(int index, const TypeDescriptor& type, voi
 
     case TYPE_HLL:
     case TYPE_OBJECT: {
-        buf_ret = _row_buffer->push_null();
+        if (_output_object_data) {
+            const StringValue* string_val = (const StringValue*)(item);
+
+            if (string_val->ptr == nullptr) {
```
Comment (morningman) [inline be/src/runtime/mysql_result_writer.cpp:121]: using string_val->is_null?

File: docs/.vuepress/sidebar/zh-CN.js [hunk 2]
```diff
@@ -636,8 +636,9 @@ module.exports = [
         "java-format-code",
         "cpp-format-code",
         "How-to-Share-blogs",
-	"commit-format-specification",
-		"minidump"
+        "commit-format-specification",
+        "minidump",
+        "bitmap-hll-file-format",
```
Comment (morningman) [inline docs/.vuepress/sidebar/zh-CN.js:641]: No same order as en doc?

**Comment by wangbo** [comment]
In terms of ```Spark Load```, fe-common would be packaged into a ```spark-dpp.jar```  and upload to Spark when submit job, So move some Class to fe-common should not affect ```Spark Load```.

**Comment by morningman** [comment]
> In terms of `Spark Load`, fe-common would be packaged into a `spark-dpp.jar` and upload to Spark when submit job, So move some Class to fe-common should not affect `Spark Load`.

Ok, thanks for your confirm

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**